### PR TITLE
semantic markup for main events listing

### PR DESCRIPTION
**Pull request by jasonkiss**
Notably made each month’s list of events an unordered list. Each month
is its own section with aria-labelledby referencing the id of the
child h2. That id is the name of the month, which should be fine as
long as we don’t list events more than 12 months in advance.
Also cleaned up extraneous attributes on main.

### CODE REVIEW

File: themes/bento/public/stylesheets/styles.css.map [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
...Aa,EAAE,QAAQ;EACvB,WAAW,EAAE,MAAM;;;;AAK3B,oEAAiB;EACb,aAAa,EAAE,cAAoB;;;AAGvC,kFAAU;;EAGN,MAAM,EAAE,IAA6B;EACrC,WAAW,EAHA,IAAiC;;;AAMhD,eAAU;;EAGN,WAAW,EZ3DH,iBAAS;EY4DjB,SAAS,EZ5CE,IAAI;EY6Cf,MAAM,EAAE,IAA8B;EACtC,WAAW,EALA,IAAiC;;;AAQhD,aAAc;EAGV,IAAI,EAAE,CAAC;EACP,UAAU,EAAE,MAAM;EAClB,KAAK,EZ7BI,QAAoB;;;AYgCjC,cAAe;EAGX,IAAI,EZnCK,QAAoB;EYoC7B,UAAU,EAAE,MAAM;EAClB,KAAK,EZpCK,QAAoB;;;AYuClC,cAAe;EAIX,IAAI,EAAE,QAA0B;EAChC,KAAK,EZ3CK,SAAqB;;;AY8CnC,eAAgB;EAIZ,IAAI,EAAE,SAAyC;EAC/C,KAAK,EZlDM,SAAqB;;;AYqDpC,eAAgB;EAIZ,IAAI,EAAE,SAAyD;EAC/D,KAAK,EZzDM,SAAqB;;;AY4DpC,kBAAmB;EAIf,IAAI,EAAE,SAAyE;EAC/E,aAAa,EAAE,IAAI;EACnB,UAAU,EAAE,KAAK;EACjB,KAAK,EZlES,SAAqB;;;AYqEvC,cAAe;EACX,OAAO,EAAE,KAAK;;EACd,MAAM,EAAE,IAAI;EACZ,KAAK,EAAE,IAAI;;;AAGf,yBAAiC;EAC/B,SAAU;IACN,MAAM,EZ3EQ,KAAgB;;;EY8ElC,aAAc;IACV,KAAK,EZ7EQ,EAAE;;;EYgFnB,cAAe;IACX,IAAI,EZjFS,EAAE;IYkFf,KAAK,EZjFS,EAAE;;;EYoFpB,cAAe;IACX,aAAa,EAAE,IAAI;IACnB,IAAI,EAAE,GAAsC;IAC5C,KAAK,EZtFS,GAAG;;;EYyFrB,eAAgB;IACZ,aAAa,EAAE,IAAI;IACnB,IAAI,EAAE,GAA2D;IACjE,KAAK,EZ3FU,GAAG;;;EY8FtB,eAAgB;IACZ,IAAI,EAAE,GAAqC;IAC3C,GAAG,EZ9GK,IAAI;IY+GZ,KAAK,EZhGU,GAAG;;;EYmGtB,kBAAmB;IACf,IAAI,EAAE,GAA2D;IACjE,GAAG,EZpHK,IAAI;IYqHZ,KAAK,EZrGa,GAAG",
```
Comment (Aupajo) [inline themes/bento/public/stylesheets/styles.css.map:3]: We should probably add maps to the `gitignore`…

File: themes/bento/public/stylesheets/_events.scss [hunk 2]
```diff
@@ -17,6 +17,21 @@
     border-bottom: 1px solid $secondary;
 }
 
+/*-------------------------------------*\
+    #MONTH OF EVENTS
+\*-------------------------------------*/
+.month-events {
```
Comment (jamesmacfie) [inline themes/bento/public/stylesheets/_events.scss:23]: Sorry, I'm going to be anal here and ask you to change a couple of things:
- Can we use camel case like the rest of the classes (`monthEvents`)
- Can we nest the ul inside the month events class (will output the same, just keeps things within context in the sass file)

**Comment by jasonkiss** [comment]
No worries. I love _what you are being_.

Happy to camelCase, and happy to make that a convention (presumably for both class and ids), but just to be anal, see: https://github.com/Aupajo/calendar-bento/search?p=2&q=class%3D%22.%2B-&type=Code&utf8=%E2%9C%93

:)

Re nesting ul inside month events class, not sure I understand. The ul is already nested within the section with class="month-events" (or what will be class="monthEvents").

And the real noob question: What's the best way to sort out changes to a PR already submitted? Close the PR, fix my changes, and resubmit? Ta.

**Comment by jamesmacfie** [comment]
Whoops, my nesting comment was confusing. I mean the `scss` file will have the `ul` nested like this:

``` scss
.monthEvents {
  padding: 0;

  ul {
    list-style: none;
    margin: 0;
    padding: 0;
  }
}

```

What I like about that is it gives context. Plus you don't have the class name written out twice.

PR question: Yeah. Close the PR, pull down the latest and sort out any conflicts that appear, resubmit.

**Pull request closed without merging**